docs/UserGuide.md

```diff
@@ -32,7 +32,7 @@ features are available in BTBB. For more in depth information on the available f
 If you have any questions regarding BTBB, check out [FAQ](#5-faq) for answers to frequently asked questions.
 
 [Disclaimers](#6-disclaimers) contains general disclaimers for our app such as restrictions on inputs and
-additional details regarding the command format. **It is important to read this section before reading the rest of the
+additional details regarding the command format. **It is important for you to read this section before reading the rest of the
```

Comment:
I don't think the user should read all of the disclaimers before reading the rest of the UG.

Reply:
This is intended